ts/features/authentication/activeSessionLogin/screens/cie/ActiveSessionLoginCieCardReaderScreen.tsx

```diff
@@ -0,0 +1,607 @@
+        case "PIN Locked":
+        case "ON_CARD_PIN_LOCKED":
+        case "ON_PIN_ERROR":
+          setError({
+            eventReason: event.event,
+            navigation: () =>
+              navigation.navigate(AUTHENTICATION_ROUTES.MAIN, {
+                screen: AUTHENTICATION_ROUTES.CIE_WRONG_PIN_SCREEN,
+                params: {
+                  remainingCount:
+                    event.event === "ON_CARD_PIN_LOCKED"
+                      ? 0
+                      : event.attemptsLeft
+                }
+              })
+          });
+          break;
+        // CIE is Expired or Revoked
+        case "CERTIFICATE_EXPIRED":
+        case "CERTIFICATE_REVOKED":
+          setError({
+            eventReason: event.event,
+            navigation: () =>
+              navigation.navigate(AUTHENTICATION_ROUTES.MAIN, {
+                screen: AUTHENTICATION_ROUTES.CIE_EXPIRED_SCREEN
+              })
+          });
+          break;
+        default:
+          break;
+      }
+    },
+    [choosenTool, navigation, setError]
+  );
+
+  const handleCieError = useCallback(
+    (error: Error) => {
+      // trackLoginCieCardReadingError();
+      handleSendAssistanceLog(choosenTool, error.message);
+      setError({ eventReason: "GENERIC", errorDescription: error.message });
+    },
+    [choosenTool, setError]
+  );
+
+  const startCie = useCallback(
+    async (useCieUat: boolean) => {
+      cieManager.removeAllListeners();
+      cieManager.onEvent(handleCieEvent);
+      cieManager.onError(handleCieError);
+      cieManager.onSuccess(handleCieSuccess);
+      cieManager.enableLog(isDevEnv);
+      cieManager.setCustomIdpUrl(useCieUat ? getCieUatEndpoint() : null);
+      await cieManager.setPin(ciePin);
+      cieManager.setAuthenticationUrl(authorizationUri);
+
+      const startOptions = {
+        readingInstructions: I18n.t(
+          "authentication.cie.card.iosAlert.readingInstructions"
+        ),
+        moreTags: I18n.t("authentication.cie.card.iosAlert.moreTags"),
+        readingInProgress: I18n.t(
+          "authentication.cie.card.iosAlert.readingInProgress"
+        ),
+        readingSuccess: I18n.t(
+          "authentication.cie.card.iosAlert.readingSuccess"
+        ),
+        invalidCard: I18n.t("authentication.cie.card.iosAlert.invalidCard"),
+        tagLost: I18n.t("authentication.cie.card.iosAlert.tagLost"),
+        cardLocked: I18n.t("authentication.cie.card.iosAlert.cardLocked"),
+        wrongPin1AttemptLeft: I18n.t(
+          "authentication.cie.card.iosAlert.wrongPin1AttemptLeft"
+        ),
+        wrongPin2AttemptLeft: I18n.t(
+          "authentication.cie.card.iosAlert.wrongPin2AttemptLeft"
+        ),
+        genericError: I18n.t("authentication.cie.card.iosAlert.genericError")
+      };
+
+      try {
+        if (Platform.OS === "ios") {
+          await cieManager.start(startOptions);
+        } else {
+          await cieManager.start();
+        }
```

Comment:
```suggestion
        await cieManager.start(Platform.OS === "ios" ? startOptions : undefined);
```
Do you think it could work written this way? Or maybe `undefined` could cause it to crash?